helper-cli/src/main/kotlin/commands/packagecuration/CreateCommand.kt

```diff
@@ -56,27 +57,32 @@ internal class CreateCommand : CliktCommand(
     override fun run() {
         val outputFile = getSplitCurationFile(outputDir, packageId, FileFormat.YAML.fileExtension)
 
-        if (outputFile.exists()) {
-            throw UsageError("File ${outputFile.absolutePath} already exists.")
+        val curations = if (outputFile.exists()) {
+            outputFile.readValue<MutableSet<PackageCuration>>()
+        } else {
+            mutableSetOf()
         }
 
+        if (packageId in curations.map { it.id }) {
+            throw IllegalArgumentException(
+                "Curation for ${packageId.toCoordinates()} already exists in ${outputFile.absolutePath}."
```

Comment:
Nit: We usually single-quote paths (and also package coordinates) in case they contain spaces. Similar below.

Reply:
Still missing here...